### ISSUE TITLE
Support extruded polygon

### ISSUE THREAD

**Issue by chris-little** [opengeospatial/ogc-feat-geo-json#33]
In Section 7.3, there is a highlighted Note:
`Should a 3D geometry that represents a simple solid constructed using an extruded polygon also be supported? This would consist of a (horizontal) 2D polygon and separate attributes for the lower and upper limits. ....`
I propose that the added complexity of an extruded polygon (perhaps a better name: `prism`) be supported, as this is supported in the API-EDR for many of the query types, such as a point or polygon. The API-EDR does not specify an extruded line string ( a `curtain`) but could easily add it. Prisms are undoubtedly useful for environmental data. 
And it is clearly the same 'pattern' for the temporal extent of a feature, and could be used for other dimensions too, such as spectral wavelength.
Many cases would only need one vertical extent/limit, as the other may be implied/defaulted by the polygon's plane of origin, such as the surface.

**Comment by cportele** [comment]
Thanks for the feedback - and for pointing out that "prism" is the correct term.

**Comment by cportele** [comment]
Meeting 2022-02-07: We agreed to add support for prism (and clarify the term), prisms seem to be use frequently. @pvretano will work on a proposal.

**Comment by cportele** [comment]
Meeting 2022-04-04: @pvretano started to investigate the topic and common ways to represent extruded polygons / prisms. Different approach are used, which can also be quite complex. How complex cases do we want to support.

The main idea was to have a simple encoding for the simple cases, where a Polyhedron may be considered "overkill", e.g. a simple LoD 0 building. Something like:

```json
{
  "type": "Prism",
  "coordinates": [ some Polygon (or Circle) geometry ],
  "lower": 500,
  "upper": 1000 
}
```

In the past, IETF defined a GML encoding for prisms in their next-gen 911 work. It is now an OGC Best Practice. This can be a good basis for our work, too. Carl will send a link.

**Comment by pvretano** [comment]
02-MAY-2022: Closed by PR #49.
